lib/widgets/wallet_list.dart

```diff
@@ -133,24 +133,24 @@ class WalletListState extends State<WalletList> {
   Widget _buildWalletItem(String walletId) {
     final theme = Theme.of(context);
     final extendedTheme = theme.extension<ExtendedTheme>()!;
-    final isSelectedWallet = walletId == selectedWallet?.id;
-    String? balance = database.walletStorage.wallets[walletId]!
-        .balanceNanoWit()
-        .availableNanoWit
-        .toString();
-    String? address = database
-        .walletStorage
-        .wallets[walletId]
-        ?.externalAccounts[selectedAddressList?[walletId] != null
-            ? int.parse(selectedAddressList?[walletId]!.split('/').last)
-            : 0]
-        ?.address
-        .toString();
     final textStyle = TextStyle(
         fontFamily: 'NotoSans',
         color: WitnetPallet.white,
         fontSize: 14,
         fontWeight: FontWeight.normal);
+    final isSelectedWallet = walletId == selectedWallet?.id;
```

Comment:
Is there a difference between having it before or after the `textStyle`definition?